### PR TITLE
Create web mart from ga4

### PR DESCRIPTION
**Pull request by Frydafly**
# Pull Request for dbt-arc-functions

- [x] If you've created a macro, make sure it is documented by running the notebook: `process_model_documentation_codegen_output.ipynb` util in this repo

- [x] Test branch on a development branch of an existing client (ideally the one that raised the bug). Do this by changing revision of the package to the branch name in `packages.yml` file in dbt State client below.
bsd-nyccfb

- [x] If new macro files were created or updated, re-run `create_or_update_standard_models.py` for the client dbt project, replacing models and dependencies

- [x] check in dbt that client package successfully runs `dbt deps`, `dbt compile`, and `dbt run`; screenshot this page
Screenshot 2023-04-04 at 2 47 01 PM

- [x] query the resulting staging model in bigquery (or by previewing mart in dbt cloud, or your IDE); screenshot this
Screenshot 2023-04-04 at 2 47 54 PM

- [x] If exists, link bug issue and jira tickets to PR
https://bluestate.atlassian.net/browse/ARC-1200?atlOrigin=eyJpIjoiMDljYmE2YTFlZjkxNDQ5MzllNzIxZmIxYjRlMzlkNzAiLCJwIjoiaiJ9

### CODE REVIEW

**Comment by Frydafly** [comment]
A few things I wanted to note for @dmbluestate & @ryantimjohn -- this is a PR for two things at this point: 

1. creating a web mart for GA4 to bigquery (one that is a work in progress, that runs!, and I'll keep adding to it as we iterate with the client NYCCFB)
2. I made a new notebook that somewhat duplicates the efforts of the codegen one, but it starts off by generating YML files that match the SQL macros you specify, and then it looks at the schema yml in your client documentation and populates those empty yml files accordingly. it worked well for me and I felt I needed these steps to make the process less of a headache. maybe we can talk about joining the two or making something better soon?

File: utils/process_model_documentation_codegen_output.ipynb
Comment (ryantimjohn) [inline utils/process_model_documentation_codegen_output.ipynb:1]: @Frydafly I think you should revert the commits on this notebook

File: utils/process_create_doc_yml_and_populate_with_codegen.ipynb [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#enter filepath for codegen output\n",
+    "codegen_output = \"/Users/frydaguedes/Projects/bsd-nyccfb/documentation/schema.yml\""
```
Comment (ryantimjohn) [inline utils/process_create_doc_yml_and_populate_with_codegen.ipynb:72]: @Frydafly I think you should remove the client name here and just have like a `LOCATION_OF_SCHEMA.YML` variable or something.

File: utils/process_create_doc_yml_and_populate_with_codegen.ipynb
Comment (ryantimjohn) [inline utils/process_create_doc_yml_and_populate_with_codegen.ipynb:1]: @Frydafly can you clear all the cell outputs and then recommit this notebook?
Screenshot 2023-04-05 at 5 47 53 PM

**Comment by ryantimjohn** [comment]
@Frydafly would you be opposed to removing the commented lines in these SQL files? I think it would go a long way towards cleaning these files up.

File: utils/process_create_doc_yml_and_populate_with_codegen.ipynb (hunk 1 above)
Comment (Frydafly) [reply utils/process_create_doc_yml_and_populate_with_codegen.ipynb:72]: oh yeah oops.

Comment (Frydafly) [reply utils/process_create_doc_yml_and_populate_with_codegen.ipynb]: done

File: utils/process_create_doc_yml_and_populate_with_codegen.ipynb
Comment (Frydafly) [reply utils/process_create_doc_yml_and_populate_with_codegen.ipynb:1]: done

File: utils/process_model_documentation_codegen_output.ipynb
Comment (Frydafly) [reply utils/process_model_documentation_codegen_output.ipynb:1]: ok, this should be done too!

**Comment by Frydafly** [comment]
> @Frydafly would you be opposed to removing the commented lines in these SQL files? I think it would go a long way towards cleaning these files up.

you're right! I will.

**Review by ryantimjohn — APPROVED** [review]
LGTM!

**Pull request merged**